pkg/applyconfiguration/testdata/cronjob/api/v1/applyconfiguration/api/v1/associativetype.go

```diff
@@ -15,7 +15,6 @@ type AssociativeTypeApplyConfiguration struct {
 func AssociativeType() *AssociativeTypeApplyConfiguration {
 	return &AssociativeTypeApplyConfiguration{}
 }
-func (b AssociativeTypeApplyConfiguration) IsApplyConfiguration() {}
```

Comment:
These removals look like a bug but are actually correct, this is happening because this pulls https://github.com/kubernetes/kubernetes/pull/132584 in

The marker is still present in the root struct: https://github.com/kubernetes-sigs/controller-tools/blob/291e3f92a10ed4cc4786de91ff579d84e9cc3a7f/pkg/applyconfiguration/testdata/cronjob/api/v1/applyconfiguration/api/v1/cronjob.go#L30